File: Capstone_Interview_Tracking_App/backend/src/main/java/com/capstone/interviewtracker/constants/messages/AuthMessages.java
package com.capstone.interviewtracker.constants.messages;

public final class AuthMessages {

    public static final String LOGIN_SUCCESS = "Login successful";
    public static final String SIGNUP_SUCCESS = "Account created successfully";
    public static final String PASSWORD_SET_SUCCESS = "Password set successfully";

    public static final String EMAIL_ALREADY_EXISTS = "Email already exists";
    public static final String INVALID_PASSWORD = "Invalid password";

    public static final String ACCOUNT_DEACTIVATED = "Your account has been deactivated. Please contact HR.";

    public static final String PASSWORD_NOT_SET = "Please set your password before logging in.";

    public static final String PASSWORD_LINK_INVALID = "Invalid or unknown password setup link.";

    public static final String PASSWORD_LINK_EXPIRED = "This password setup link has expired. Please request a new one.";

    public static final String USER_NOT_FOUND = "User not found";

    public static final String SET_PASSWORD_LINK_REQUIRED = "Please set your password using the link sent to your email before logging in.";

    public static final String INVALID_OR_UNKNOWN_PASSWORD_LINK = "Invalid or unknown password setup link.";

    public static final String PASSWORD_LINK_ALREADY_USED = "This password setup link has already been used.";

    public static final String NO_USER_FOR_TOKEN = "No user found for this token.";

    private AuthMessages() {
    }
}

File: Capstone_Doctor_Appointment_Booking_System/backend/constants/auth_messages.py
"""
Authentication messages.
"""


class AuthMessages:
    INVALID_CREDENTIALS = "Invalid email or password"
    ACCOUNT_PENDING_APPROVAL = "Your application is still pending admin approval."
    ACCOUNT_REJECTED = "Your application was rejected. Please contact support."
    ACCOUNT_INACTIVE = "This account is not active. Please contact support."
    TOKEN_EXPIRED = "Your session has expired. Please log in again."
    INVALID_TOKEN = "Invalid authentication token."
    SETUP_LINK_INVALID = "This setup link is invalid or has already been used"
    SETUP_LINK_EXPIRED = (
        "This setup link has expired. Please ask an admin to re-approve your application."
    )
    INSUFFICIENT_ROLE = "This action requires one of these roles: {roles}"